frontend/micro-ui/web/micro-ui-internals/packages/libraries/src/services/elements/User/index.js

```diff
@@ -106,16 +106,22 @@ export const UserService = {
     return roles && Array.isArray(roles) && roles.filter((role) => accessTo.includes(role.code)).length;
   },
 
-  changePassword: (details, stateCode) =>
-    ServiceRequest({
+  changePassword: async (details, stateCode) => {
+    const data = await ServiceRequest({
```

Comment:
can you show a toast , say that password updated and say session will be closed please login with new password

Reply:
keep some 5 sec delay